libpod/runtime_ctr.go

```diff
@@ -915,6 +915,44 @@ func (r *Runtime) evictContainer(ctx context.Context, idOrName string, removeVol
 	return id, cleanupErr
 }
 
+// DependMap populates of map of all containers depending on the specified container
```

Comment:
```suggestion
// DependMap populates a map of all containers depending on the specified container
```

Reply:
Comment seems unrelated to the actual function?

Reply:
Fixed